### PR TITLE
feat-alert

### PR DESCRIPTION
**Pull request by abhij1607**
image
simple alert component

### CODE REVIEW

**Review by GuptaEesh — COMMENTED** [review]
Comment updated. Please follow through.

File: components/Alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
+.alert-container {
+  box-shadow: 0 0 1px rgb(0 0 0 / 10%), 0 2px 6px rgb(0 0 0 / 10%);
+  padding: 10px;
+  max-width: 30rem;
+}
+
+.alert {
+  padding: 1rem;
```
Comment (shivampandey0) [inline components/Alert/alert.css:8]: try to use variables.

**Review by shivampandey0 — COMMENTED** [review]
Try to use variables for colors and units as much as possible. 
It'll help with consistency and responsiveness.

File: components/Alert/alert.css (hunk 1 above)
Comment (abhij1607) [reply components/Alert/alert.css:8]: thanks for the suggestion, will try to use it later on

File: components/Alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  margin-bottom: 1rem;
+  border: 1px solid transparent;
+  border-radius: 5px;
+}
```
Comment (GuptaEesh) [inline components/Alert/alert.css:12]: Use prettier for spacing. It will make your code consistent.

**Review by GuptaEesh — COMMENTED** [review]
Apart from the suggestion added, the code looks clean. Good to merge.

**Pull request merged**